src/sentry/api/urls.py

```diff
@@ -1756,6 +1759,11 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         OrganizationMemberInviteDetailsEndpoint.as_view(),
         name="sentry-api-0-organization-member-invite-details",
     ),
+    re_path(
+        r"^(?P<organization_id_or_slug>[^\/]+)/reinvite-member/(?P<member_invite_id>[^\/]+)/$",
```

Comment:
This URL pattern doesn't seem to be following any convention. Is there a better way to do this?